### ISSUE TITLE
Conflicts with the installed package 'bat'

### ISSUE THREAD

**Issue by DO1JLR** [gantsign/ansible_role_bat#62]
I got this error on  Debian GNU/Linux 11 (bullseye)
```yml
fatal: [example.com]: FAILED! => changed=false 
  invocation:
    module_args:
      allow_change_held_packages: false
      allow_downgrade: false
      allow_unauthenticated: false
      autoclean: false
      autoremove: false
      cache_valid_time: 0
      clean: false
      deb: /home/ansible/.ansible/tmp/downloads/bat-musl_0.22.1_amd64.deb
      default_release: null
      dpkg_options: force-confdef,force-confold
      fail_on_autoremove: false
      force: false
      force_apt_get: false
      install_recommends: null
      lock_timeout: 60
      only_upgrade: false
      package: null
      policy_rc_d: null
      purge: false
      state: present
      update_cache: null
      update_cache_retries: 5
      update_cache_retry_max_delay: 12
      upgrade: null
  msg: Conflicts with the installed package 'bat'
```

**Comment by DO1JLR** [comment]
F figured out, there is this package installed that is causing thees trouble:
```
bat/stable,stable,now 0.12.1-6+b2 amd64 [installed]
  cat(1) clone with syntax highlighting and git integration
```
#64 will remove it once installation failed.

**Comment by freemanjp** [comment]
Fixed in version [1.16.1](https://github.com/gantsign/ansible_role_bat/releases/tag/1.16.1).
